pmp-reactjs-ui/src/pages/dashboard/Dashboard.js

```diff
@@ -18,6 +19,7 @@ import authServiceIcon from '../../svg/auth_services_icon.svg';
 function Dashboard() {
   const navigate = useNavigate();
   const { t } = useTranslation();
+  const arabicLang = RTLStyles(getUserProfile().langCode);
```

Comment:
remove arabicLang variable name
rename to isLoginLanguageRTL in all pages

Reply:
Done